react/features/base/react/components/web/Watermarks.js

```diff
@@ -319,27 +221,52 @@ class Watermarks extends Component<Props, State> {
  * Maps parts of Redux store to component prop types.
  *
  * @param {Object} state - Snapshot of Redux store.
+ * @param {Object} ownProps - Component's own props.
  * @returns {Props}
  */
-function _mapStateToProps(state) {
+function _mapStateToProps(state, ownProps) {
     const { isGuest } = state['features/base/jwt'];
-    const { customizationReady, logoClickUrl, logoImageUrl } = state['features/dynamic-branding'];
-    const { room } = state['features/base/conference'];
+    const {
+        customizationReady,
+        customizationFailed,
+        defaultBranding,
+        useDynamicBrandingData,
+        logoClickUrl,
+        logoImageUrl
+    } = state['features/dynamic-branding'];
+    const welcomePageVisible = !state['features/base/conference'].room;
+    const {
+        DEFAULT_LOGO_URL,
+        JITSI_WATERMARK_LINK,
+        SHOW_JITSI_WATERMARK,
+        SHOW_JITSI_WATERMARK_FOR_GUESTS,
+        filmStripOnly
+    } = interfaceConfig;
+    let _showJitsiWatermark = (!filmStripOnly
+          && (customizationReady && !customizationFailed)
+          && (SHOW_JITSI_WATERMARK || (isGuest && SHOW_JITSI_WATERMARK_FOR_GUESTS)))
+    || welcomePageVisible;
+    let _logoUrl = logoImageUrl;
+    let _logoLink = logoClickUrl;
+
+    if (useDynamicBrandingData) {
+        if (isVpaasMeeting(state)) {
```

Comment:
Does this mean that people using their own deployments and relaying on dynamic branding will no longer be able to use it this way? We know people are already using this, dynamic branding should apply always IMHO.

Reply:
No. Everything should work as before. 

For `vpaas` meetings though, we don't show the logo if it was not explicitly set or the request has failed (these were _specs_ I received).

Reply:
Ack